pages/es/learningcenter/mellon-on-the-markets.html

```diff
@@ -13,6 +13,14 @@
 		</div>
 		<div class="Page-Grid__grid-wrap">
 		    <div class="Page-Grid__grid-wrap__box">
+				<a href="{{site.url}}/ebooks/ebook_en_MellonOnTheMarketSep2016.pdf">
```

Comment:
Hey :)
thanks for reply. I will try this but I am not sure this will fix the issues I have. You see, I was using baseurl at first, but than Travis was complaining being unable to resolve url. So I had to change all files on blog to use url instead of baseurl. That fixed the problem. 

Anyways, let me change this as you suggested and will see what happens.

Thanks